Source/santactl/Commands/SNTCommandSync.m

```diff
@@ -32,7 +32,7 @@ @implementation SNTCommandSync
 #pragma mark SNTCommand protocol methods
 
 + (BOOL)requiresRoot {
-  return YES;
+  return NO;
```

Comment:
Is there any reason not to remove the call to `DropRootPrivileges` below? We don't do this for other commands that don't require root

Reply:
Some commands have a reason to run as root even if it's not required (e.g. fileinfo) whereas there is no benefit to running this as root and due to the XPC work it does I think it's safer if it drops privileges before it makes any connections.

Reply:
Missed this blurb in the description:
```
We still ensure that privileges are dropped, just in case someone does execute as root.
```
But i guess i'm not sure the harm?

Reply:
Fair enough. We may want to consider doing this on additional commands as well then (e.g. `status`, maybe `version`)